components/_twig-components/functions/pl_attach-library.function.php

```diff
@@ -5,5 +5,20 @@
  */
 
 $function = new Twig_SimpleFunction('attach_library', function ($string) {
-  return $string;
-});
+  // Get Library Name from string.
+  $libraryName = substr($string, strpos($string, "/") + 1);
+
+  // Search dist root directory for library.
+  $files = glob('dist/*/' . $libraryName . '/' . $libraryName . '*.js');
```

Comment:
Do we care about the situations where $libraryName is different from the file name?

Reply:
actually, this code needed to be refactored. The file name is covered by `*.js` so the library name just has to match the name of the component.